docs/src/less/ds4/index-large.less

```diff
@@ -8,7 +8,7 @@
     border-right: 1px solid #ececec;
     height: 100%;
     overflow-y: scroll;
-    padding: 0 16px 64px 16px;
+    padding: 24px 16px 64px;
```

Comment:
I presume these DS4 changes are to accommodate the new ds6 header/toast thingie?

Reply:
Yup.